apache/trafficcontrol
# Adding Types table and details

This PR is not related to any issue. It adds the Types table and details views to Traffic Portal V2.

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

Make sure all the unit/ e2e tests pass.
Manual testing:
Run TPV2 
Go to the Types page
Make sure everything looks good
Make sure all the right click functionality works
Verify that all the CRUD functionality works for Types 

## If this is a bugfix, which Traffic Control versions contained the bug?

- master

## PR submission checklist
